presto-docs/src/main/sphinx/release/release-0.277.rst

```diff
@@ -0,0 +1,27 @@
+=============
+Release 0.277
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Add UDF :func:`find_first` to find the first array element which matches a predicate.
+* Fix compilation error in common subexpression elimination when duplicate expressions are present.
+* Fix enforcement of query analyzer timeout during planning.
+* Add support to fail query when number of leaf plan nodes exceeds a limit. This can be controlled with ``leaf_node_limit_enabled`` and ``max_leaf_nodes_in_plan`` session properties.
```

Comment:
order is Fixes first and then Additions.
https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-changes